src/components/tl/mlx5/mcast/tl_mcast_context.c

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include <inttypes.h>
+#include "tl_mcast.h"
+#include "utils/arch/cpu.h"
+#include <ucs/sys/string.h>
+#include "src/core/ucc_service_coll.h"
+#include "tl_mlx5.h"
+
+ucc_status_t ucc_tl_mlx5_mcast_runtime_progress(void *mcast_coll_context)
```

Comment:
why we need this function?

Reply:
in the latest fixes for nonblocking mcast progress, we no longer use this function. so Im gonna remove it. thanks for pointing it out